src/api/services/format-date-service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import moment from 'moment';
+
+@Injectable()
+export class FormatDate {
+
+    constructor() {}
+
```

Comment:
I like the setLang and getLang methods, I think they might be encapsulated into an interface, Something like LanguageAware or LanguageSupport interface.